recOrder/calib/calibration_workers.py

```diff
@@ -405,16 +405,19 @@ def _save_bg_recon(self, bg_path: StrOrBytesPath):
         else:
             os.mkdir(bg_recon_path)
         # save raw reconstruction to zarr store
-        writer = WaveorderWriter(save_dir=bg_recon_path)
-        writer.create_zarr_root(name="bg_reconstruction")
-        rows, columns = self.birefringence.shape[-2:]
-        writer.init_array(
-            position=0,
-            data_shape=(1, 2, 1, rows, columns),
-            chunk_size=(1, 1, 1, rows, columns),
-            chan_names=["Retardance", "Orientation"],
-        )
-        writer.write(self.birefringence[:2], p=0, t=0, z=0)
+        with open_ome_zarr(
+            bg_recon_path + "bg_reconstruction",
```

Comment:
What is the intended file name here? I'm getting this file structure for a default background acquisition:
```text
│   calibration_metadata.txt
│   gui_state.yml
│   State0.tif
│   State1.tif
│   State2.tif
│   State3.tif
│   State4.tif
│
├───reconstruction
│       orientation.png
│       retardance.png
│
└───reconstructionbg_reconstruction
    │   .zattrs
    │   .zgroup
```
While the previous version produces:
```text
│   calibration_metadata.txt
│   gui_state.yml
│   State0.tif
│   State1.tif
│   State2.tif
│   State3.tif
│   State4.tif
│
└───reconstruction
    │   orientation.png
    │   retardance.png
    │
    └───bg_reconstruction.zarr
```
So maybe this?
```suggestion
            os.path.join(bg_recon_path, "reconstruction"),
```